java/src/main/java/org/example/SimpleConsumer.java

```diff
@@ -0,0 +1,40 @@
+package org.example;
+
+import org.apache.pulsar.client.api.*;
+
+import java.util.concurrent.ExecutionException;
+
+public class SimpleConsumer {
+    public static String PULSAR_URL = "";
+    public static String PULSAR_JWT = "";
```

Comment:
@charvakcpatel007 @deltasquare4 can we update this config same as done for go 
in both `consumer & producer` files
https://github.com/pandio-com/code-samples/blob/master/go/simple-producer-consumer/consumer.go#L13-L14
```
const (
		pulsarURL        string = "{{ pulsar_url }}"
		pulsarJWT        string = "{{ pulsar_jwt }}"
		pulsarTopic      string = "persistent://public/default/topic-go-pandio-go"
		subscriptionName string = "my-subscription-name"
	)
```